### PR TITLE
[VERY WIP] Add pets list

### PR DESCRIPTION
**Pull request by yuindustries**

### CODE REVIEW

File: internal/proc/procfs.go [hunk 1]
```diff
@@ -115,6 +115,13 @@ func (f ProcFS) mapProcs(mapFn func(PetsProc) PetsProc) error {
 
 // Read all the procs from the JSON file
 func (f ProcFS) ProcsFromFS() ([]PetsProc, error) {
+	f.mu.Lock()
```
Comment (pmt) [inline internal/proc/procfs.go]: Why was this added?

**Comment by nicks** [comment]
@pmt let's do a code walkthru on Monday to talk thru how it fits together! 

When i first added the mutex Locks, I thought that we were going to do a lot of concurrency. But right now we don't do any concurrency at all. maybe we should remove them!

**Comment by nicks** [comment]
discussed with patricia offline, we're going to leave the mutexes for now!
